src/Compilers/CSharp/Desktop/CommandLine/CommandLineParser.cs

```diff
@@ -28,12 +28,12 @@ internal CSharpCommandLineParser(bool isInteractive = false)
         protected override string RegularFileExtension { get { return ".cs"; } }
         protected override string ScriptFileExtension { get { return ".csx"; } }
 
-        internal sealed override CommandLineArguments CommonParse(IEnumerable<string> args, string baseDirectory, string additionalReferencePaths)
+        internal sealed override CommandLineArguments CommonParse(IEnumerable<string> args, string baseDirectory, string sdkDirectory, string additionalReferenceDirectories)
         {
-            return Parse(args, baseDirectory, additionalReferencePaths);
+            return Parse(args, baseDirectory, sdkDirectory, additionalReferenceDirectories);
         }
 
-        public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string baseDirectory, string additionalReferencePaths = null)
+        public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string baseDirectory, string sdkDirectory, string additionalReferenceDirectories = null)
```

Comment:
Add some doc comments please. `string`, `string`, `string` is easy to call incorrectly.